backends/sqs/backend.go

```diff
@@ -389,9 +395,17 @@ func (s *Backend) GetMessages(ctx context.Context, in *rpc.GetMessagesRequest) (
 		for key, valueStruct := range message.MessageAttributes {
 			attributes[key] = *valueStruct.StringValue
 		}
+
+		sqsAttributes := make(map[string]string)
```

Comment:
create the map only if in.RequireQueueSystemAttributes? 
Not sure what happens if sqsAttributes is an empty map